src/matlab2tikz.m

```diff
@@ -3710,6 +3710,9 @@ function warnFacetedInterp(m2t)
     str = sprintf('\\addplot[%s] plot table[%s] {%s};\n', ...
                  opts_print(m2t, drawOptions, ','), ...
                  opts_print(m2t, tabOpts, ','), table);
+
+    [m2t, baseline] = drawBaseline(m2t,h,isHoriz);
```

Comment:
What bothers me a bit is that you pass `isHoriz` to a parameter called `isVertical`. I guess `isHoriz` means the bars are in a horizontal direction, so the base line should be vertical.
I think that should become apparent from the code as well (if my explanation is correct, of course).